workos/directory_sync.py

```diff
@@ -121,6 +127,7 @@ def list_directories(
             limit (int): Maximum number of records to return. (Optional)
             before (str): Pagination cursor to receive records before a provided Directory ID. (Optional)
             after (str): Pagination cursor to receive records after a provided Directory ID. (Optional)
+            organization: ID of an Organization (Optional)
```

Comment:
I think in the docstring we can keep it in the same position, since it makes more sense having it first (and it doesn't have any effect on compatibility).

Reply:
👍🏽